_data/students.yml

```diff
@@ -1,562 +1,569 @@
-  blog: meets2tushar.wordpress.com
-  bio: 15, Indian, Love HTML/CSS, FOSS
-
-- name: Yashvardhan Sharma
-  github: yvsja
-  image: yashvardhan_sharma.jpg
-  facebook: yash.vardhan.18062
-
-- name: Hesham Burawi
-  github: Hesham-Burawi
-  image: Hesham-Burawi.jpeg
-  twitter: Hesham_Burawi
-  bio: 13, Libyan/British, Love Sleep, Food, Coding
-
-- name: Zahari Asparuhov
-  github: zahariasp
-  image: zahari.jpg
-  facebook: zari.asparuhov
-  bio: 17, Bulgaria, web developing and web servers with NodeJS
-
-- name: Arpna kumari
-  github: arpnakumari
-  facebook: arpna kumari
-  twitter: arpna07
-  image: arpna.jpg
-  blog: arpnablog.wordpress.com
-  bio: A 16 yr old from INDIA ,wants to pursue medical field.
-
-- name: Gautam Jajoo
-  github: gautamjajoo
-  facebook: Gautam Jajoo
-  twitter: gautamjajoo
-  image: gautam.jpg
-  blog: gautamjajoo.wordpress.com
-  bio: A 14 year old Indian, whose passion is Computer Science.
-
-- name: Shivam Singh
-  twitter: shivamkumar4848
-  image: shivam.png
-  bio: A 16 year old Indian, study in XI, like playing.
-
-- name: Guneet Singh
-  github: Guneetconvent2002
-  image: Guneet.jpg
-  facebook: Guneet
-  bio: A 14 year old boy who wants to do somthing for the nation.
-
-- name: Vishal Singh
-  github: vishu15singh
-  image: Vishalsingh.jpg
-  twitter: vishu15singh
-  bio: 16, Student, Footballer, Indian.
-
-- name: Srijan Jha
-  github: SrijanJha28
-  image: SrijanJha.jpg
-  twitter: SrijanJha2
-  bio: 16, Indian, coder, hacker, tech enthusiast
-
-- name: Vivek Kumar
-  github: vivekkumarsvm
-  image: vivek.jpg
-  twitter: vivekkumarsvm46
-  bio: 16, Indian, Computer Science Student
-
-- name: Peirce Chow
-  github: peircechow
-  image: peircechow.jpg
-  facebook: peirce.chowzhengjie
-  twitter: peircechow
-  bio: 17, Singaporean, aspiring coder
-
-- name: Koh Yi Ting
-  github: kohyt
-  image: kohyt.jpg
-  twitter: msft007
-  bio: 17, Singaporean, learning Python
-
-- name: Samuel Chua Jia Cong
-  github: Samuel Chua
-  image: samuelchua.jpg
-  twitter: scjc2001
-  facebook: samuel.cong
-  blog: samuelchuadhs.wordpress.com
-  bio: 15, Singaporean, Cuber, Explorer
-  lat: 1.3950886
-  lng: 103.9014255
-
-- name: Prajwal K Patil
-  github: PrajwalPatill
-  image: prajwal.jpg
-  twitter: p_kailasanath
-  blog: https://prajwalpatil.tech.blog
-  bio: I am cool coder.
-  facebook: prajwal.udchankar
-  lat: 15.849695299999999
-  lng: 74.4976741
-
-- name: Muktiadi Akhmad Januar
-  github: Muktiadi
-  image: Muktiadi.jpg
-  twitter: Zukaizen
-  facebook: MuktiadiAkhmadJanuar
-  bio: Love to learning from basics.
-  lat: -6.895726746997539
-  lng: 107.80893713235855
-
-- name: Guo Jialu Benjamin
-  github: benjaminguo
-  image: benjaminguo.jpg
-  twitter: guoJLbenjamin
-  bio: Cynical
-  lat: 1.3553794
-  lng: 103.86774439999999
+# New members please add your credentials to a random position in the file.
+# This is to remove conflicts with other people adding their credentials.
+
+- name: Prince John

```

Comment:
Can't understand why are you doing this again and again, please don't add carriage returns nor modify the entire page